### ISSUE TITLE
Can't resolve worker_threads & SSR styling

### ISSUE THREAD

**Issue by stephenjason89** [nuxt/framework#4251]
### Environment

------------------------------
- Operating System: `Darwin`
- Node Version:     `v16.13.1`
- Nuxt Version:     `2.16.0-27358576.777a4b7f`
- Package Manager:  `yarn@1.22.17`
- Builder:          `webpack`
- User Config:      `srcDir`, `ssr`, `dev`, `alias`, `generate`, `typescript`, `plugins`, `components`, `buildModules`, `modules`, `vuetify`, `build`, `bridge`
- Runtime Modules:  `-`
- Build Modules:    `@nuxtjs/vuetify@1.12.3`, `@nuxt/bridge@3.0.0-27467056.93c4dfe`
------------------------------

### Reproduction

https://github.com/stephenjason89/NUXTREPRODUCTION
@danielroe I tried to remove most of the unnecessary stuff here. Please let me know if this is okay. 

Thanks

### Describe the bug

<img width="1115" alt="image" src="https://user-images.githubusercontent.com/3810177/162714959-6b87057f-4a8d-4bf8-9307-d0571eb04bcd.png">

SSR
<img width="636" alt="image" src="https://user-images.githubusercontent.com/3810177/162715831-9ce2feba-2a93-464b-9178-5991207b451a.png">

After hydration
<img width="897" alt="image" src="https://user-images.githubusercontent.com/3810177/162715899-36aacc13-f05b-4a3f-bb12-80b88f51725e.png">

### Additional context

I get a console warning regarding worker_threads

This warning is not present on nuxt 2.15 before migrating to nuxt-bridge

Please also notice when page is rendered with SSR then before hydration, all vuetify styling is not present. When it hydrates then vuetify styling is applied.

It's not like this on regular nuxt 2.15

**Comment by danielroe** [comment]
The styling is a separate issue - see https://github.com/nuxt/framework/issues/1207.

Thanks for the reproduction; I've made it a bit smaller here: https://stackblitz.com/edit/github-puxrjq.

I believe the issue is that the version of `jspdf` you are using is depending on an old version of `fflate`, and the node version is being imported in the browser build of nuxt. I've added a workaround in the linked repro above.
